### DISCUSSION TITLE
Why does the metrics spec recommend `count` instead of `total` for counter metric suffixes?

### DISCUSSION THREAD

**Discussion by emschwartz** [open-telemetry/opentelemetry-specification#3304] [Q&A]
It's probably too late to change this but does anyone know why `count` was chosen instead of `total`?

I understand that compatibility with OpenMetrics is not necessarily a goal of the OpenTelemetry project but using a different synonym for something like this seems like an unnecessary divergence. (OpenMetrics requires the [`_total` suffix](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#suffixes).)

For context, I'm working on a library that makes it easy to instrument functions with metrics and then writes queries for you to understand the generated data. We have an ongoing discussion here about [using OpenTelemetry or OpenMetrics semantic conventions](https://github.com/orgs/autometrics-dev/discussions/4) and there are some quite small divergences that force us to make more of a choice than would be ideal.

**Comment by trask** [comment]
@open-telemetry/wg-prometheus wdyt?

for reference, here's the current [OTel spec recommendation to use count](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#use-count-instead-of-pluralization)

**Comment by emschwartz** [comment]
I opened an issue to propose changing this here: https://github.com/open-telemetry/opentelemetry-specification/issues/3457
